### PR TITLE
Adding Anthos on GCP and K8s questions

### PR DESCRIPTION
**Pull request by cloudgrimm**
Adding more sections on K8s with core concepts and scheduling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4468,18 +4468,35 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
...
+
+The control plane component kube-scheduler asks the following questions,
+1. What to schedule? It tries to understand the pod-definition specifications 
+2. Which node to schedule? It tries to determine the best node with available resources to spin a pod
+3. Binds the Pod to a given node
+
+View more [here](https://www.youtube.com/watch?v=rDCWxkvPlAw)
 </b></details>
 
+<<<<<<< HEAD
```
Comment (bregman-arie) [inline README.md:4491]: Mistake?

File: README.md [hunk 2]
```diff
@@ -4468,18 +4468,35 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
...
+<details>
+<summary> How are labels and selectors used?</summary><br><br>
+</details>
+
+#### Basic Commands
+=======
 #### Kubernetes Commands
+>>>>>>> master
 
 <details>
```
Comment (bregman-arie) [inline README.md:4501]: Same here

File: README.md [hunk 3]
```diff
@@ -7575,6 +7592,60 @@ startap-script
...
+of consistency of their legacy applications in a hybrid or multicloud world. From this explanation the core ideas can be drawn from these statements;
+
+* Managed -> the customer does not need to worry about the underlying software intergrations, they just enable the API.
+* application platform -> It consists of open source tools like K8s, Knative, Istio and Tekton
+* Enterprises -> these are usually organisations with complex needs 
+* Consistency -> to have the same policies declaratively initiated to be run anywhere securely e.g on-prem, GCP or other-clouds (AWS or Azure)
+</b></details>
+
+<details>
+<summary>List the technical components that make up Anthos?</summary><b><b>
```
Comment (bregman-arie) [inline README.md:7609]: No need to add a question mark at the end. It's not a question but rather a request.

File: README.md [hunk 4]
```diff
@@ -7575,6 +7592,60 @@ startap-script
...
+<summary>Which load balancing options are available?</summary><b><b>
+
+* Networking load balancing for L4 and HTTP(S) Load Balancing for L7 which are both managed services that do not require
+  additional configuration. 
+* Ingress for Anthos which allows the ability to deploy a load balancer that serves an application across multiple clusters 
+  on GKE
+</b></details>
+
+<details>
+<summary>Can you deploy Anthos on AWS?</summary><b><b>
```
Comment (bregman-arie) [inline README.md:7643]: No really interesting question in my opinion. What do you think?

File: README.md [hunk 5]
```diff
@@ -4468,18 +4468,35 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
 </b></details>
 
 <details>
-<summary>What services types are there?</summary><br><b>
+<summary> What does being cloud-native mean?</summary><br><br>
 </b></details>
 
 <details>
-<summary>Explain Liveness probe</summary><br><b>
+<summary> Explain the pet and cattle approach of infrastructure with respect to kubernetes?</summary><br><br>
```
Comment (bregman-arie) [inline README.md:4475]: No need for a question mark

File: README.md [hunk 6]
```diff
@@ -4468,18 +4468,35 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
...
 </b></details>
 
+#### Scheduling
+
 <details>
-<summary>Explain Readiness probe</summary><br><b>
```
Comment (bregman-arie) [inline README.md:4479]: Why did you remove this question?

File: README.md [hunk 7]
```diff
@@ -4468,18 +4468,35 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
 </b></details>
 
 <details>
-<summary>What services types are there?</summary><br><b>
+<summary> What does being cloud-native mean?</summary><br><br>
 </b></details>
 
 <details>
-<summary>Explain Liveness probe</summary><br><b>
```
Comment (bregman-arie) [inline README.md:4475]: Why did you remove this question?

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (CloudGrimm) [reply README.md:4475]: It was a mistake, I think when I forked to my local repo it had issues

File: README.md (hunk 4 above)
Comment (CloudGrimm) [reply README.md:7643]: So Anthos is meant to run on other clouds, it is the core of the service, having it stay there is necessary. For instance soon it will run on Azure

File: README.md (hunk 6 above)
Comment (CloudGrimm) [reply README.md:4479]: It was a mistake

**Review by cloudgrimm — COMMENTED** [review]
Have done the changes as requested

File: README.md [hunk 8]
```diff
@@ -4468,17 +4468,41 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
 </b></details>
 
 <details>
-<summary>What services types are there?</summary><br><b>
+<summary>What services types are there?</summary><br><b>	
 </b></details>
 
 <details>
 <summary>Explain Liveness probe</summary><br><b>
-</b></details>
```
Comment (bregman-arie) [inline README.md:4476]: Mistake?

File: README.md [hunk 9]
```diff
@@ -4468,17 +4468,41 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
...
+</b><details>
 
 <details>
 <summary>Explain Readiness probe</summary><br><b>
 </b></details>
 
+<summary> What does being cloud-native mean</summary><br><br>
```
Comment (bregman-arie) [inline README.md:4482]: missing question mark + double <br>

**Review by bregman-arie — CHANGES_REQUESTED** [review]

**Review by cloudgrimm — COMMENTED** [review]
requested changes made

File: README.md [hunk 10]
```diff
@@ -4479,6 +4479,31 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
 <summary>Explain Readiness probe</summary><br><b>
 </b></details>
 
+<details>
+<summary> What does being cloud-native mean?</summary><br><br>
```
Comment (bregman-arie) [inline README.md:4483]: switch <br><br> with <br><b>

File: README.md [hunk 11]
```diff
@@ -4479,6 +4479,31 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
...
+</b></details>
+
+<details>
+<summary> Explain the pet and cattle approach of infrastructure with respect to kubernetes</summary><br><br>
```
Comment (bregman-arie) [inline README.md:4487]: switch <br><br> with <br><b>

File: README.md [hunk 12]
```diff
@@ -4479,6 +4479,31 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
...
+</b></details>
+
+#### Scheduling
+
+<details>
+<summary> How does scheduling work in kubernetes?</summary><br><br>
```
Comment (bregman-arie) [inline README.md:4493]: switch <br><br> with <br><b>

File: README.md [hunk 13]
```diff
@@ -7575,6 +7600,60 @@ startap-script
 <summary>What the following commands does? `gcloud deployment-manager deployments create`</summary><br><b>
 </b></details>
 
+#### Anthos
+
+<details>
+<summary> What is Anthos</summary><b><b>
```
Comment (bregman-arie) [inline README.md:7606]: switch <b><b> with <br><b>

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: README.md (hunk 10 above)
Comment (CloudGrimm) [reply README.md:4483]: Please explain further I am a bit confused?

Comment (bregman-arie) [reply README.md]: Sorry, it's the html formatting.
What I originally wrote is "switch \</summary>\<br>\<br> with \</summary>\<br>\<b>"

**Comment by bregman-arie** [comment]
@CloudGrimm please switch every \</summary>\<br>\<br> with \</summary>\<br>\<b> like in every other question in this repo

**Comment by CloudGrimm** [comment]
@bregman-arie have done the edits

File: README.md [hunk 14]
```diff
@@ -7616,6 +7616,12 @@ startap-script
 <summary>What the following commands does? `gcloud deployment-manager deployments create`</summary><br><b>
 </b></details>
 
+#### Google Kubernetes Engine (GKE)
+
+<details>
+<summary>What is GKE</summary><br><br>
```
Comment (bregman-arie) [inline README.md]: Missed that one

File: README.md [hunk 15]
```diff
@@ -7669,6 +7675,16 @@ instances in the project.
 * Yes, Anthos on AWS is now GA. For more read [here](https://cloud.google.com/anthos/gke/docs/aws)
 </b></details>
 
+<details>
+<summary>List and explain the enterprise security capabilities provided by Anthos</summary><b><b>
```
Comment (bregman-arie) [inline README.md]: Missed here as well

File: README.md (hunk 14 above)
Comment (CloudGrimm) [reply README.md]: Please check again because that was rectified long back

**Comment by CloudGrimm** [comment]
@bregman-arie all of the changes have been done long back

**Comment by bregman-arie** [comment]
@CloudGrimm thanks for the effort on this one

**Comment by CloudGrimm** [comment]
> @CloudGrimm thanks for the effort on this one

@bregman-arie thanks for the feedback, love this repo and its now my go to for DevOps quick reference. Hope it continues to grow.
